{"name": "test_login_failed[case1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert '' == '账号或密码不正确'\n  - 账号或密码不正确", "trace": "self = <testcase.test_admin_login.TestLogin object at 0x0000021D070A9B40>\nadmin_manage_driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"c22057c828b17a55e9e7a1d3c53deb06\")>\ncase = {'check': '账号或密码不正确', 'code': 'lemon', 'passwd': '123456a123', 'title': '密码错误', ...}\n\n    @pytest.mark.usefixtures(\"admin_manage_driver\")\n    @pytest.mark.usefixtures(\"admin_refresh_page\")\n    @pytest.mark.parametrize(\"case\", error_datas)\n    def test_login_failed(self, admin_manage_driver, case):\n        driver = admin_manage_driver\n        LoginPage(driver).login(case.get(\"user\"), case.get(\"passwd\"), case.get(\"code\"))\n>       assert LoginPage(driver).get_errormsg_from_top_popup() == case.get(\"check\")\nE       AssertionError: assert '' == '账号或密码不正确'\nE         - 账号或密码不正确\n\ntestcase\\test_admin_login.py:39: AssertionError"}, "attachments": [{"name": "stderr", "source": "6c523c9d-405f-4896-aebf-15c9a1287bec-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'title': '密码错误', 'user': 'student', 'passwd': '123456a123', 'code': 'lemon', 'check': '账号或密码不正确'}"}], "start": 1674740970202, "stop": 1674740972234, "uuid": "19a19908-30e8-476a-a7bf-cb1cfb86aa05", "historyId": "7ccc6c3f3b46f126454c96049f35acee", "testCaseId": "d9546ea4e5187a7a337ce2e7838ce24f", "fullName": "testcase.test_admin_login.TestLogin#test_login_failed", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('admin_refresh_page')"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_admin_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "linjie"}, {"name": "thread", "value": "6844-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_admin_login"}]}